website/orders/models.py

```diff
@@ -100,7 +100,9 @@ class Product(models.Model):
     name = models.CharField(max_length=50, unique=True, null=False, blank=False)
     icon = models.CharField(
         max_length=20,
-        unique=True,
+        unique=False,
```

Comment:
If this isn't unique anymore, maybe we should also show the product name in the shift admin then. Because now we only display the customer name and icon. If the icon isn't unique we might get ambiguity

Reply:
Hmm, I had some troubles with the icons. What should we give all soda cans for example? A different one each? The admin panel will now show a product name if no icon is provided. I can do two things:

1. Always show the product name (but then we might have to do some CSS trickery for the layout)
2. Make this unique again (but then the soda cans can't all have the same icon)

I prefer 1. What do you think?

Reply:
Let's go for 1. (and maybe if the CSS is too hard, we could add a shortname field to products and only display that)

Reply:
This might be resolved in the latest commit.